tests/test-shell-scripts.sh

```diff
@@ -17,7 +17,9 @@
 set -o errexit
 set -o nounset
 
-OUTPUT_FOLDER="$(mktemp -d)"
+OUTPUT_FOLDER="$(pwd)/tmp/output"
```

Comment:
Is this change needed?

Reply:
Yes, it looks like When running `docker run -v`, only subfolders under current folder are accessible to the container on MacOS runner.

Tried to `chmod`, `--privileged`, none of these works.

`OUTPUT_FOLDER="$(mktemp -d)"` works if run on a Macbook, but not MacOS runner